### ISSUE TITLE
[BUG] Static broken on Google Images

### ISSUE THREAD

**Issue by mcandre** [darkreader/darkreader#2941]
### **Describe the bug**

Static is broken for Google Images. Which is unfortunate, because Dynamic is broken on other websites and the two Filter modes slow down JS event handling.

### **Steps to reproduce**

- Set to Static mode
- Go to Google Images
- Search for "dogs"

### **Expected behavior**

Dog images appear.

### **Actual behavior**

The images are not visible.

### **System Information:**
 - OS: Android 10
 - Browser: Firefox latest stable
 - Dark Reader version: Latest stable

**Comment by Gusted** [comment]
I have 0 explaination even after debugging.

**Comment by refael722** [comment]
You can use "Only for #####" on a Google site and then set up a dynamic filter, this will only apply to the specific site.

**Comment by Myzel394** [comment]
One dirty solution would be to set `display: unset` on the outer `a` tag of the `img`s - this worked for me at least. Maybe we should add that, so that people can at least use the static engine somehow?

**Comment by arche-dev** [comment]
> One dirty solution would be to set `display: unset` on the outer `a` tag of the `img`s - this worked for me at least. Maybe we should add that, so that people can at least use the static engine somehow?

So like:

```html
<a src="https://somerandomsite.com">
	<img src="https://dogs.com/images/dog323827.png">
</a>
```

```CSS
a {
   display: unset !important;
}

**Comment by Myzel394** [comment]
@arche-dev yes exactly but not on the sidebar images. I mean the currently non-visible images.
